choose_your_own_adventure/jsonParser/jsonParser.go

```diff
@@ -0,0 +1,44 @@
+package jsonParser
+
+import (
+	"bufio"
+	"encoding/json"
+	"io"
+	"os"
+
+	"github.com/gopher-training/choose_your_own_adventure/structs"
+)
+
+//ParseArcStories parses our json file containing arc stories into a slice of structs.ArcStory
+func ParseArcStories(fileName string) (map[string]structs.ArcStory, error) {
+	fileJSON, err := os.Open("gopher.json")
```

Comment:
always check for **err** right after you received this object

Reply:
Fixed.